.github/workflows/build-docs-verifier.yml

```diff
@@ -0,0 +1,40 @@
+name: Build docs-verifier
```

Comment:
@gewarren Is GitHub Actions disabled for this repository? This doesn't seem to run.

Reply:
This is not a docs repo, so we don't need to enable this action. Also, actions are currently disable in this repo.

Reply:
@gewarren This yml file doesn't enable this action, and verifies the action itself builds and run few unit tests.

Reply:
Where you interested in doing "dog-fooding" @Youssef1313, for testing purposes?

Reply:
Gotcha. I've enabled actions now so it should run.

Reply:
> Where you interested in doing "dog-fooding" @Youssef1313, for testing purposes? I think that is a valid use case to enable them and helps quickly show consumers how they're intended to be used.

Will add dogfooding yml too, in addition to this one.

Reply:
> Where you interested in doing "dog-fooding" @Youssef1313, for testing purposes? I think that is a valid use case to enable them and helps quickly show consumers how they're intended to be used.

Will add dogfooding yml too, in addition to this one.

Reply:
> Gotcha. I've enabled actions now so it should run.

@gewarren For some reason it doesn't run

Reply:
I'm out of ideas. At one point I had "allow all actions" enabled, and it still didn't run. Maybe @IEvangelist can figure it out.

Reply:
Oh actually, now that it's pull_request_target, it won't run until it's in the base branch.